### DISCUSSION TITLE
Which LSP to use with Lua?

### DISCUSSION THREAD

**Discussion by paaguti** [orbitalquark/textadept#389] [Q&A]
I'm getting errors when I edit Lua code (for example, init.lua) The message is related with the LSP.
Which LSP do I need for Lua?

Thx, /PA

**Comment by orbitalquark** [comment]
What errors are you getting? Without any configuration, the LSP module 
uses its own Lua language server for Lua files like your 
*~/.textadept/init.lua*.
